UI/src/app/home/home.component.css

```diff
@@ -38,9 +38,8 @@
     margin: 250px auto 0 auto;
 }
 .display-container{
-    width: 840px;
+    width: 700px;
     height: 560px;
-    border: solid white 2px;
```

Comment:
if you enlarge the container, you need to scroll to see the error message, I would have kept it like that

I also thought the box looked nice, so i kept it